README-zh-cn.md

```diff
@@ -0,0 +1,172 @@
+<h1 align="center" style="clear: both;">
+    <img src="docs/images/graphar-logo.svg" width="350" alt="GraphAr">
+</h1>
+<p align="center">
+    一种用于图数据存储和读取的开源标准数据文件格式。
+</p>
+
+[![GraphAr
+CI](https://github.com/apache/incubator-graphar/actions/workflows/ci.yml/badge.svg)](https://github.com/apache/incubator-graphar/actions)
+[![Docs
+CI](https://github.com/apache/incubator-graphar/actions/workflows/docs.yml/badge.svg)](https://github.com/apache/incubator-graphar/actions)
+[![GraphAr
+Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://graphar.apache.org/docs/)
+[![Good First
+Issue](https://img.shields.io/github/labels/apache/incubator-graphar/Good%20First%20Issue?color=green&label=Contribute)](https://github.com/apache/incubator-graphar/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```

Comment:
Maybe we can add an icon to indicate that there are other version languages of README? Same as English README.